lib/minfraud/components/credit_card.rb

```diff
@@ -94,8 +114,9 @@ def validate
         return if !Minfraud.enable_validation
 
         validate_telephone_country_code('bank_phone_country_code', @bank_phone_country_code)
-        validate_regex('issuer_id_number', /\A[0-9]{6}\z/, @issuer_id_number)
-        validate_regex('last_4_digits', /\A[0-9]{4}\z/, @last_4_digits)
+        validate_regex('issuer_id_number', /\A(?:[0-9]{6}|[0-9]{8})\z/, @issuer_id_number)
+        validate_regex('last_digits', /\A(?:[0-9]{2}|[0-9]{4})\z/, @last_digits)
+        validate_regex('last_4_digits', /\A(?:[0-9]{2}|[0-9]{4})\z/, @last_4_digits)
```

Comment:
Do we need this line? I think we don't have the variable now so it won't do anything.

Reply:
Ah nope. I guess my revert wasn't sufficient since I had changed this based on your suggestion before https://github.com/maxmind/minfraud-api-ruby/pull/60/commits/8f2af774e44cc115d2e9c0981a95b776ecc17333